flutter_keyboard_visibility/android/src/main/java/com/jrai/flutter_keyboard_visibility/FlutterKeyboardVisibilityPlugin.java

```diff
@@ -5,6 +5,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.view.ViewTreeObserver;
+import androidx.core.view.WindowInsetsCompat;
```

Comment:
androidx is a dependancy right? You were able to add this without changing the gradle file?

Last time I tried bringing AndroidX it caused issues for a lot of users, so I reverted it.